encoder_config_test.go

```diff
@@ -108,12 +164,10 @@ func TestJSONEncoder_EncodeEntry(t *testing.T) {
 				zap.String("foo", "bar"),
 				zap.Int("count", 8),
 			}
+
+			ecsCfg := ECSCompatibleEncoderConfig(tc.cfg)
 			//parse config and create encoder from it
```

Comment:
There's no parsing happening any more, and I think the fact that we're creating an encoder is pretty clear on the following line. Delete comment?